pkgs/by-name/ma/material-maker/package.nix

```
@@ -0,0 +1,140 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  autoPatchelfHook,
+  copyDesktopItems,
+  makeDesktopItem,
+  godot3-export-templates,
+  godot3-headless,
+  alsa-lib,
+  libGL,
+  libGLU,
+  libpulseaudio,
+  libX11,
+  libXcursor,
+  libXext,
+  libXfixes,
+  libXi,
+  libXinerama,
+  libXrandr,
+  libXrender,
+  zlib,
+  udev,
+}:
+stdenv.mkDerivation rec {
+  pname = "material-maker";
+  version = "1.3";
+
+  src = if stdenv.isLinux then fetchurl {
+    url = "https://github.com/RodZill4/material-maker/releases/download/1.3/material_maker_1_3_linux.tar.gz";
+    hash = "sha256-Y8+ZwXy3zqnsxqqaZeVgFSxLzmUkq+rBzbq8tEDc8/g=";
+  } else throw "unsupported platform";
```

Use `fetchzip` instead of `fetchurl` since it auto-unpacks. We can also reuse the defined attributes like `pname`, `version` and `versionUnderscore`

```suggestion
  pname = "material-maker";
  inherit version;

  src =
    if stdenv.isLinux then
      fetchzip {
        url = "https://github.com/RodZill4/${pname}/releases/download/${version}/material_maker_${versionUnderscore}_linux.tar.gz";
        hash = "sha256-WEu5gVfnswB5zYzu3leOL+hKOBzJbn48gHQKshlfOh4=";
      }
    else
      throw "unsupported platform";
```